apps/bajour/components/bajour/best-of-wepublish/best-of-wepublish.tsx

```diff
@@ -0,0 +1,131 @@
+import {styled} from '@mui/material'
+import {ApiV1, BuilderTeaserGridBlockProps} from '@wepublish/website'
+
+import {BestOfWePublishTeasers} from './best-of-wepublish-teasers'
+
+export type BestOfWePublishProps = Omit<BuilderTeaserGridBlockProps, 'teasers'> & {
+  teasers?: Array<ApiV1.PeerArticleTeaser>
+}
+
+const BestOfWePublishWrapper = styled('div')`
+  grid-column: 1/13;
+  margin: 2rem 0;
+  display: grid;
+  grid-gap: 2rem;
+  padding: 0 ${({theme}) => theme.spacing(2)};
+
+  ${({theme}) => theme.breakpoints.up('md')} {
+    padding: 0;
+  }
+`
+
+const Wrapper = styled('div')`
+  display: grid;
+  gap: ${({theme}) => theme.spacing(2)};
+  grid-template-columns: repeat(2, 1fr);
+  align-items: stretch;
+
+  ${({theme}) => theme.breakpoints.up('md')} {
+    grid-template-columns: repeat(3, 1fr);
+  }
+`
+
+const BestOfWePublishHeader = styled('div')`
+  width: 100%;
+  background-color: #ffbaba;
+`
+
+const HeaderText = styled('span')`
+  background-color: ${({theme}) => theme.palette.common.white};
+  color: #ffbaba;
+  font-size: 1.4rem;
+  font-weight: bold;
+  margin-left: 20%;
+  padding: 0 ${({theme}) => theme.spacing(2)};
+  display: inline-block;
+
+  ${({theme}) => theme.breakpoints.up('md')} {
+    font-size: 2rem;
+  }
+`
+
+const BestOfWePublishFooter = styled('div')`
```

Comment:
There's a margin bottom missing as the button is now hanging into neighbouring elements. While the margin exists on the parent, that's not really portable and prone to errors.